packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -1251,18 +1324,34 @@ class Calendar<
   _animateShowView(): void {
     // @ts-expect-error ts-error
     fx.stop(this._view.$element(), true);
-    this._popAnimationView(this._view, POP_ANIMATION_FROM, POP_ANIMATION_TO, ANIMATION_DURATION_SHOW_VIEW);
+    this._popAnimationView(
+      this._view,
+      POP_ANIMATION_FROM,
+      POP_ANIMATION_TO,
+      ANIMATION_DURATION_SHOW_VIEW,
+    );
 
     const { viewsCount } = this.option();
 
     if (viewsCount > 1) {
       // @ts-expect-error ts-error
       fx.stop(this._additionalView.$element(), true);
-      this._popAnimationView(this._additionalView, POP_ANIMATION_FROM, POP_ANIMATION_TO, ANIMATION_DURATION_SHOW_VIEW);
+      this._popAnimationView(
+        this._additionalView,
+        POP_ANIMATION_FROM,
+        POP_ANIMATION_TO,
+        ANIMATION_DURATION_SHOW_VIEW,
+      );
     }
   }
 
-  _popAnimationView(view, from, to, duration) {
+  _popAnimationView(
+    view: MonthView | YearView | DecadeView | CenturyView,
+    from: AnimationConfig['from'],
+    to: AnimationConfig['to'],
+    duration: number,
+  ): Promise<unknown> {
+    // @ts-expect-error ts-error
     return fx.animate(view.$element(), {
```

Comment:
Try:
```suggestion
    return fx.animate(view.$element().get(0), {
```

Reply:
it doesn't seem safe since fx functions are expecting jq el accordind to the code. it's just an issue with type

Reply:
my bad, issue only actual for `isAnimation`. Will update all such places in my next pr